front/src/modules/simulationResult/components/SpaceTimeChart/SpaceTimeChart.tsx

```diff
@@ -51,11 +51,13 @@ export type SpaceTimeChartProps = {
   selectedProjection?: OsrdSimulationState['selectedProjection'];
   simulation?: SimulationSnapshot;
   simulationIsPlaying?: boolean;
+  timeScaleDomain?: TimeScaleDomain;
   isDisplayed?: boolean;
   onSetBaseHeight?: (newHeight: number) => void;
   dispatchUpdateSelectedTrainId: DispatchUpdateSelectedTrainId;
   dispatchPersistentUpdateSimulation: DispatchPersistentUpdateSimulation;
   setTrainResultsToFetch?: (trainSchedulesIDs?: number[]) => void;
+  setTimeScaleDomain?: React.Dispatch<React.SetStateAction<TimeScaleDomain>>;
```

Comment:
I think it's easier to read like that

```suggestion
  setTimeScaleDomain?: (newTimeScaleDomain: TimeScaleDomain) => void;
```

Reply:
Yes agree, thanks ! I had to use React.Dispatch earlier during developments, but it's not required anymore, good catch ! :muscle: